scripts/add_extra_components.py

```diff
@@ -120,6 +116,8 @@ def attach_stores(n, costs, elec_opts):
             capital_cost=costs.at["hydrogen storage underground", "capital_cost"],
         )
 
+        n.madd("Carrier", ["H2 electrolysis", "H2 fuel cell"])
```

Comment:
Is this changed? Used to be "H2 Electrolysis" and "H2 Fuel Cell". If changed, this needs to be mentioned in release notes, but judging from the code below, it should still be capitalised.

Reply:
It hadn't changed. More like not exist in the first place. It was my early attempt to add the carrier individually. Since there is already `sanitize_carriers(n, snakemake.config)` in the bottom, removing all of the new carrier won't make much differences.